aeacus-src/utility.go

```diff
@@ -1,21 +1,43 @@
 package main
 
 import (
-	"fmt"
 	"io/ioutil"
 	"os"
+	"strconv"
 	"time"
 )
 
+var verboseEnabled = false
 var aeacusVersion = "1.1.1"
 
+// writeFile wraps ioutil's WriteFule function, and prints
+// the error the screen if one occurs.
 func writeFile(fileName string, fileContent string) {
 	err := ioutil.WriteFile(fileName, []byte(fileContent), 0644)
 	if err != nil {
-		fmt.Println(err)
+		failPrint("Error writing file: " + err.Error())
```

Comment:
Can we please use `log.Fatalln(errors.Wrap(err, "Error writing file: " + fileName))` instead here since it's not returning anything?

Reply:
I think `log.Fatalln` will call `os.Exit(1)`, which isn't necessarily desired, since we shouldn't quit if `phocus` or `aeacus` fail to write to `ScoringReport.html` or something similar. Ideally I think `writeFile` should just print an error and let the program go about its business. We're also not using `log` anywhere else (could be a different line of effort to add that though?).

Reply:
Ah ok. I think I replaced all the places with stuff like `panic(err)` and `fmt.Println(err)` + `os.Exit(1)` with `log.Fatalln(errors.Wrap(err, "msg"))`, but I get how it's not appropriate in this situation.